sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample3_ExtractKeyPhrasesWithWarnings.cs

```diff
@@ -18,23 +18,33 @@ public void ExtractKeyPhrasesWithWarnings()
 
             var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
-            string document = "Anthony runs his own personal training business so thisisaverylongtokenwhichwillbetruncatedtoshowushowwarningsareemittedintheapi";
+            string document = @"Anthony runs his own personal training business so
+                              thisisaverylongtokenwhichwillbetruncatedtoshowushowwarningsareemittedintheapi";
```

Comment:
I wonder if [pneumonoultramicroscopicsilicovolcanoconiosis](https://en.wikipedia.org/wiki/Longest_word_in_English) can trigger a warning.

Reply:
wow! sadly, no :( the warning get's triggered with more than 64 characters